config/manager/deployment.yaml

```diff
@@ -40,4 +40,10 @@ spec:
         env:
         - name: CLI_CACHE_DIR
           value: /var/cache/kubectl
+        livenessProbe:
+          httpGet:
+            path: /debug/health
+            port: 8080
+          failureThreshold: 1
+          periodSeconds: 10
```

Comment:
Add an `initialDelaySeconds` so that the liveness probe is not racing with the manager startup.

Reply:
Added a 10 second delay. Let me know if you think that's enough.